cmd/secret-wrapper/main.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"os/signal"
+	"syscall"
+
+	coreclientset "k8s.io/client-go/kubernetes/typed/core/v1"
+
+	"github.com/openshift/ci-tools/pkg/util"
+)
+
+const (
+	DIR = "/tmp/secret"
```

Comment:
nit: this is not idomatic Go

Reply:
I'll admit it took me a long time to realize the capitalization was the problem (had to `grep` for `const`) =)